app/templates/events/view/tickets/index.hbs

```diff
@@ -95,11 +95,11 @@
       <tr>
         {{#link-to 'events.view.tickets.orders.index' tagName='td'
         class='ui green inverted segment link'}}
-          <span>{{t 'Completed'}}</span>
```

Comment:
@harshitagupta30 Can you confirm this from the server guys? Will there be stats for `completed` or not ?